common/src/main/templates/io/netty/util/collection/KObjectMap.template

```diff
@@ -76,48 +71,14 @@ public interface @K@ObjectMap<V> {
     V remove(@k@ key);
 
     /**
-     * Returns the number of entries contained in this map.
-     */
-    int size();
-
-    /**
-     * Indicates whether or not this map is empty (i.e {@link #size()} == {@code 0]).
-
+     * Gets an iterable collection of primitive entries contained in this map. As an optimization,
+     * the {@link PEntry}s returned by the {@link Iterator} may change as the {@link Iterator}
+     * progresses. The caller should not rely on {@link PEntry} key/value stability.
      */
-    boolean isEmpty();
-
-    /**
-     * Clears all entries from this map.
-     */
-    void clear();
+    Iterable<PEntry<V>> entries();
```

Comment:
nit: Maybe `entryIterator()` to be more consistent with the `Map` interface of `entrySet()`?